File: platform/sdk/nodejs/examples/graph/graph-007_cdc_consumer_id_rotate.js
const { GraphClient } = require("../../xuanji-sdk-graph");
const client = new GraphClient({ endpoint: "graph.local" });
client.cdcNew("consumer-old-7");
const result = client.cdcConsumerIdRotate("consumer-old-7", "consumer-new-7");
if (!result.rotated) process.exit(1);
console.log("XJ-OK: graph-007_cdc_consumer_id_rotate");
process.exit(0);

File: platform/sdk/python/examples/graph/graph-007_cdc_consumer_id_rotate.py
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from mox_sdk_graph import GraphClient

client = GraphClient({"endpoint": "graph.local"})
client.cdcNew("consumer-old-7")
result = client.cdcConsumerIdRotate("consumer-old-7", "consumer-new-7")
if not result["rotated"]:
    sys.exit(1)
print("XJ-OK: graph-007_cdc_consumer_id_rotate")
sys.exit(0)
